x-pack/plugins/log_explorer/public/components/common/translations.ts

```diff
@@ -122,22 +122,30 @@ export const flyoutShipperLabel = i18n.translate('xpack.logExplorer.flyoutDetail
   defaultMessage: 'Shipper',
 });
 
-export const flyoutHoverActionFilterForText = (text: unknown) =>
+export const actionFilterForText = (text: unknown) =>
```

Comment:
Can you help me understand why the `text` is typed as `unknown` here instead of `string`?

Reply:
Thank you for bringing this to attention. Initially this was not properly typed due to us directly using the type from `DataTableRecord['flattened']` which has a type of `flattened: Record<string, unknown>;`

https://github.com/elastic/kibana/blob/4f9e2bdf195caace9ce860230929c86e31cacb6a/packages/kbn-discover-utils/src/types.ts#L20-L37

This should not be the case anymore. I will fix it 👍🏼